src/backend/main/java/com/example/boilerscout/BoilerScoutApplication.java

```diff
@@ -54,6 +54,13 @@
         return profileController.updateProfile(body);
     }
 
+    
+    @RequestMapping(value = "/profile/get", params = {"id"}, method = RequestMethod.GET)
+    @ResponseBody
+    public Map<String, Object> getProfile(@RequestParam("id") String id,@RequestParam("token") String token, @RequestParam("goal") String goal){
```

Comment:
The parameters should just look like `@RequestParam String id` etc. You don't need the `"id"` in parenthesis. Also change `goal` to something more conventional